src/rdkafka_broker.c

```diff
@@ -923,7 +923,8 @@ rd_kafka_broker_send (rd_kafka_broker_t *rkb, rd_slice_t *slice) {
 
 
 static int rd_kafka_broker_resolve (rd_kafka_broker_t *rkb,
-                                    const char *nodename) {
+                                    const char *nodename,
+                                    rd_bool_t forceResetAddress) {
```

Comment:
We typically use snake_case in the code, so rename this to `reset_cached_addr`.

Reply:
Updated it. Thansk!